telemetry_device/KafkaConnection.cs

```diff
@@ -28,15 +30,23 @@ public KafkaConnection(TelemetryDeviceSettings telemetryDeviceSettings)
             _producer = new ProducerBuilder<Null, string>(producerConfig).Build();
             _adminClient = new AdminClientBuilder(adminConfig).Build();
         }
-        public void SendToTopic(string topicName, Dictionary<string,(int,bool)> paramDict)
+        public void SendFrameToKafka(string topicName, Dictionary<string,(int,bool)> paramDict)
```

Comment:
Unit the same code line in SendFrameToKafka and in SendStatisticsToKafka to one func